src/styles/blocks/main.scss

```diff
@@ -0,0 +1,14 @@
+.main {
+  display: grid;
+
+  @each $screenSize, $numberOfRow in $screenSize-numberOfRow {
+    @media (min-width: $screenSize) {
```

Comment:
We have a requirement:
_there should always be 4 cards in a row (not 2, 3 or 5)_

Reply:
❗️❗️❗️ if you already familiar with grid use it instead of flex and implement the [Grid Layout instead](https://github.com/mate-academy/layout_catalog/blob/master/grid.md) =>

Reply:
or maybe... do I not fully understand the task?